Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -944,6 +944,387 @@ PhysicalParticleContainer::AddPlasma (int lev, RealBox part_realbox)
     // The function that calls this is responsible for redistributing particles.
 }
 
+void
+PhysicalParticleContainer::AddPlasmaFlux (int lev, amrex::Real dt)
+{
+    WARPX_PROFILE("PhysicalParticleContainer::AddPlasmaFlux()");
+
+    const Geometry& geom = Geom(lev);
+    const amrex::RealBox& part_realbox = geom.ProbDomain();
+
+    amrex::Real num_ppc_real = plasma_injector->num_particles_per_cell_real;
+#ifdef WARPX_DIM_RZ
+    Real rmax = std::min(plasma_injector->xmax, geom.ProbDomain().hi(0));
```

Comment:
Let's include `<algorithm>` in that file for `std::min`

Reply:
This will be added in PR #1947 (since std::min is already used elsewhere).